### DISCUSSION TITLE
Game balance discussion

### DISCUSSION THREAD

**Discussion by JeodC** [DescentDevelopers/Descent3#274] [General]
Concerning game balance tweaks, the original code has a large difficulty spike from Rookie to Hotshot. For example, in `AIGame.cpp`, the robot Tubbs has the following difficulty changes:

- DIFFICULTY_TRAINEE: Increases maximum speed by `20%` (* 1.2f), maximum acceleration by `10%` (* 1.1f), and sets the full anger time to `22.0f`.
- DIFFICULTY_ROOKIE: Increases maximum speed by `30%` (* 1.3f), maximum acceleration by `50%` (* 1.5f), and sets the full anger time to `20.0f`.
- DIFFICULTY_HOTSHOT: Increases maximum speed by `70%` (* 1.7f), maximum acceleration by `150%` (* 2.5f), and sets the full anger time to `18.0f`.
- DIFFICULTY_ACE: Increases maximum speed by `75%` (* 1.75f), maximum acceleration by `180%` (* 2.8f), and sets the full anger time to `15.0f`.
- DIFFICULTY_INSANE: Increases maximum speed by `80%` (* 1.8f), maximum acceleration by `200%` (* 3.0f), and sets the full anger time to `12.0f`.

Should the base source modify the difficulty spreads to be more even, or keep them as-is? Is Tubbs the only robot affected? It looks like Homonculus and Old Scratch have more even distributions.

**Comment by SiriusTR** [comment]
I wouldn't have any concerns with doing a game balance pass, the rationale being that I doubt anyone would want to play the "status quo" version that much. Insane could possibly be left intact if we _do_ have purists out there.
I do think that a rebalance should not focus purely on smoothing out the difficulty bumps, though, but taking a more holistic look at how fun the game is to play on each difficulty level. That will likely take longer, but I still believe it's necessary. Sometimes robots are too difficult to hit on _most_ of the difficulty levels, and to fix that we'd have to wind down the speed/acceleration across the board (since making them larger is often too dangerous).
